spec/todoApp.spec.js

```diff
@@ -0,0 +1,177 @@
+const todoApp = require("../src/todoApp");
+
+describe("Todo app", () => {
+  afterEach(function () {
```

Comment:
Interesting - why did you choose afterEach v beforeEach?

Reply:
When importing todoApp object from src/todoApp, it comes with clean initial state. So when running Jasmine, the first test loads with initial values got from the import. Then the values are updated/set to the ones provided in spec setup, spec is executed and verified. After the spec/test is complete, afterEach method is called and resets the values to the default ones. I probably could use beforeEach, but it made more sense to me to reset the values after the test is run rather than before running it.